plugins/cmd/chainlink-ocr3-capability/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3"
+	consensustypes "github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/reportingplugins"
+	ocr3rp "github.com/smartcontractkit/chainlink-common/pkg/loop/reportingplugins/ocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+const (
+	loggerName = "PluginOCR3Capability"
+)
+
+func main() {
```

Comment:
Side note: are there any docs around core's config that lets it launch separate plugin processes? I saw code that reads env vars in the delegate but couldn't find actual config examples yet.

Reply:
Not as far as I'm aware 😬